docs/modules/default-architecture/pages/splitting.adoc

```diff
@@ -0,0 +1,86 @@
+= Splitting
+
+Splitting documents is a fine art, where you must navigate the tradeoff between splitting too much and splitting too little. Splitting too much will result in a large number of documents, which will increase cost. Splitting too little will result in a smaller number of documents, which will save money but also decrease the model's accuracy.
```

Comment:
The last sentence may be too much of a blanket generalization -- larger chunk sizes can result in better accuracy given your documents and the specific question since they may include more surrounding context. I would suggest rephrasing it to something like:
"""
Splitting documents into smaller segments called _chunks_ is an essential step when embedding your data into a vector store. RAG pipelines will retrieve relevant chunks to serve as context for the LLM to pull from when generating responses, which makes it important that the retrieved chunks provide the right amount of contextual information to answer the question, and no more than that. 

The difficulty here is in selecting a chunk size. Smaller chunk sizes may result in lower cost by providing fewer tokens to the context window, but the embedding may miss out on broader contextual information. Larger chunk sizes include that contextual information but may have diluted responses due to unnecessary information being included in the context. 
"""

Reply:
Added